depthai_helpers/production_support_server_api.py

```diff
@@ -61,7 +61,7 @@ def sync():
 			results = [result for result in reader]
 		
 		response = requests.post(
-			f'{STATS_SERVER_URL}/results', 
+			f'{PRODUCTION_SUPPORT_SERVER_URL}/results', 
```

Comment:
I would love to make this more general, would be nice to be able to select table, so I would change it to something like
` f'{PRODUCTION_SUPPORT_SERVER_URL}/TABLE_NAME'`

Reply:
I think multiple tables are unnecessary since you can use the `stage` field to differentiate different stages.